lib/camunda-cloud/ElementTemplatesReplaceProvider.js

```diff
@@ -0,0 +1,105 @@
+import {
```

Comment:
We could consider to move this into a sub-folder, mirroring the bpmn-js structure (does it amend the `replace` feature, if so lets move it into a `replace` folder).